tests/tests_unit/test_rules/test_models/test_conceptual_model.py

```diff
@@ -315,7 +315,7 @@ def test_dangling_properties(self, dm_dict) -> None:
             _ = VerifyAnyDataModel(validate=True).transform(input_rules)
 
         assert not issues.has_errors
-        assert len(issues) == 5
+        assert len(issues) == 4
         assert len([issue for issue in issues if issue.__class__ == DanglingPropertyWarning]) == 2
```

Comment:
Similar here, what about doing a count on the issue types and ensure that you get what is expected.